PersonApi/V1/Controllers/PersonApiController.cs

```diff
@@ -144,5 +145,15 @@ public async Task<IActionResult> UpdatePersonByIdAsync([FromBody] UpdatePersonRe
 
             return null;
         }
+
+        private void RestrictAccessToDisallowedEmail(PersonQueryObject query)
+        {
+            // This is an exceptional case where a specific user is not allowed to access a specific record
+            // See HPT-641 for more information
```

Comment:
I know it's a release PR, but I figure, at some point it would be useful to changes this to be Google Doc link instead as the Jira boards can get archived & unreachable over time.